.github/docker/Dockerfile.packaging-alma8

```diff
@@ -0,0 +1,12 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/almalinux:8
+
+RUN <<EOF
+
+dnf install -y rpm-build rpm-sign zstd
+dnf clean all
+
+EOF
```

Comment:
Why EOF ? Maybe the following code would be better 
```
RUN dnf install -y rpm-build rpm-sign zstd && \
    dnf clean all
```

Reply:
EOF makes instructions cleaner and more readable : https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/